framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/Commands/ProjectCreationCommandBase.cs

```diff
@@ -749,6 +748,59 @@ protected void ConfigureAngularJsonForThemeSelection(ProjectBuildArgs projectArg
         }
     }
 
+    protected virtual async Task ConfigureAngularAfterMicroserviceServiceCreatedAsync(ProjectBuildArgs projectArgs, string template)
+    {
+        if (!MicroserviceServiceTemplateBase.IsMicroserviceServiceTemplate(projectArgs.TemplateName))
+        {
+            return;
+        }
+
+        var rootPath = Directory.GetCurrentDirectory();
+        var uiFramework = FindMicroserviceSolutionUiFramework(rootPath);
+
+        if (uiFramework != UiFramework.Angular)
+        {
+            return;
+        }
+
+        Logger.LogInformation("Setting up the angular library...");
+
+        var libraryName = projectArgs.SolutionName.ProjectName.ToKebabCase();
+        var angularAppPath = Path.Combine(rootPath, "apps", "angular");
+
+        var result = await CreateAngularLibraryAsync(libraryName, angularAppPath);
+
+        Logger.LogInformation(result);
+    }
+
+    protected virtual async Task<string> CreateAngularLibraryAsync(
+        string libraryName,
+        string workingDirectory,
+        bool isSecondaryEndpoint = false,
+        bool isModuleTemplate = true,
+        bool isOverride = true)
+    {
+        //TODO: Can we improve this validations ?
+        if (string.IsNullOrWhiteSpace(libraryName))
+        {
+            throw new CliUsageException("Angular library name can not be empty");
+        }
+
+        if (string.IsNullOrWhiteSpace(workingDirectory))
+        {
+            throw new CliUsageException("Angular project path can not be empty");
+        }
+
+        var commandBuilder = new StringBuilder($"npx ng g @abp/ng.schematics:create-lib --package-name {libraryName}");
+
+        commandBuilder.Append($" --is-secondary-entrypoint {(isSecondaryEndpoint ? "true" : "false")} ");
+        commandBuilder.Append($" --is-module-template {(isModuleTemplate ? "true" : "false")} ");
+        commandBuilder.Append($" --override {(isOverride ? "true" : "false")} ");
```

Comment:
Normally I could add `if` statements yet, we have to set `true` or `false` to schematic argument it is required, this command hasn't default values